README.md

```diff
@@ -252,37 +254,26 @@ imported with [`go_repository`](#go_repository), will have libraries named
 
 ## Repository rules
 
-### `go_repositories`
+### `go_rules_dependancies`
 
 ``` bzl
-go_repositories(go_version)
+go_rules_dependancies()
 ```
 
 Adds Go-related external dependencies to the WORKSPACE, including the Go
 toolchain and standard library. All the other workspace rules and build rules
 assume that this rule is placed in the WORKSPACE.
+When nested workspaces arrive this will be redundant.
```

Comment:
Add link to https://bazel.build/designs/2016/09/19/recursive-ws-parsing.html

Reply:
Done